tests/queries/0_stateless/90001_array_fold.reference

```diff
@@ -0,0 +1,8 @@
+23
```

Comment:
The test name is wrong, should be somewhere around 01700. Just take the next number that is not used.